EventListener/UserContextSubscriber.php

```diff
@@ -71,6 +77,7 @@ public function __construct(
         $this->userIdentifierHeaders = $userIdentifierHeaders;
         $this->hashHeader            = $hashHeader;
         $this->ttl                   = $ttl;
+        $this->addVaryOnHash            = $addVaryOnHash;
```

Comment:
@dbu whihc is exactly why we have decided not to use alignment in Symfony :smile: